CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to this project will be documented in this file based on the
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/3.2.1...HEAD)
 
 ### Backward Compatibility Fixes
+- Use short array syntax `[]` instead of long `array()` - so break compatibility with PHP 5.3
```

Comment:
Officialy, PHP 5.3 was removed on 12. November 2015. But until now, it was still possible to use almost all of the library with PHP 5.3. In general if there was no big benefit of breaking 5.3, I didn't, but with introducing [] in all files it definitively doesn't work anymore with 5.3. That is why I mentioned it to add it. 

Following suggestion: Lets go with the suggestion from @merk as it will be confusing it "breaking 5.3" shows up again. Sorry about the confusion from my side.